selene/support/shared/config.py

```diff
@@ -219,6 +219,9 @@ def wait(self, entity):
         hook = self._inject_screenshot_and_page_source_pre_hooks(self.hook_wait_failure)
         return Wait(entity, at_most=self.timeout, or_fail_with=hook)
 
+    def reset_source(self):
+        self._source = Source()
```

Comment:
Heh... 
actually, the "self.source" is something really internal for SharedConfig to be revealed even by method like "reset_source".
While mering your PR I will refactor ir from config.reset_source to config.quit_driver
The idea behind the latter is the following... The `source` is some technical low level trick to handle shared driver in shared config. Object public methods should reflect high level api  not low level. Then, if shared config user operates "driver terms", then we have to provide some similary named methods...